addOns/authhelper/authhelper.gradle.kts

```diff
@@ -29,7 +29,16 @@ zapAddOn {
                 dependencies {
                     addOns {
                         register("client") {
-                            version.set(">=0.10.0")
+                            version.set(">=0.11.0")
+                        }
+                        register("spiderAjax") {
```

Comment:
This should not be here, this extension is for the client add-on which should not add/have any dependencies on the AJAX Spider. There's an extension above specifically for the AJAX Spider.